projects/storefrontlib/src/cms-components/product/product-interests/stock-notification/stock-notification.component.spec.ts

```diff
@@ -183,129 +154,108 @@ describe('StockNotificationComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should not load stock notificaiton data, when customer is not login', () => {
```

Comment:
Personal view, the order of the cases can be:
 - Test the display of elements when customer is login/not login, with/without channel set ....
 - Test operation such as create stock notification(loading/success/fail), remove stock notification(loading/success/fail)

Reply:
agree!!